docs/resources/watch.md

```diff
@@ -205,5 +204,5 @@ Optional:
 
 Required:
 
-- `type` (String) The type of filter, such as `regex`, `package-type` or `ant-patterns`
+- `type` (String) The type of filter, such as `regex`, `package-type` or `ant-patterns`. `package-type` works only with `all-repos` type. `ant-patterns` works only with `project`
```

Comment:
Are you enforcing this?

Reply:
Not with the built-in validation func. The description suggests it's cross-attributes validation so it will need to use the `CustomDiff` func to build our own validation. See repository.go in Artifactory provider for example.

Reply:
then it will be a different PR and a ticket